### ISSUE TITLE
Uploading video from Vimeo does not work

### ISSUE THREAD

**Issue by tmazuryk** [Cores-ts/Fundingbox-support#6]
Tried to upload this video https://vimeo.com/134601979 to this profile:http://fundingbox.com/profiles/atsora

**Comment by xurxo** [comment]
Yep, only Youtube is available right now, We'll add a message to warn the user
